src/Kunstmaan/NodeBundle/EventListener/NodeTranslationListener.php

```diff
@@ -46,22 +47,29 @@ class NodeTranslationListener
     /**
      * NodeTranslationListener constructor.
      *
-     * @param FlashBagInterface            $flashBag
+     * @param SessionInterface|FlashBagInterface $session
      * @param LoggerInterface              $logger
      * @param SlugifierInterface           $slugifier
      * @param RequestStack                 $requestStack
      * @param DomainConfigurationInterface $domainConfiguration
      * @param PagesConfiguration           $pagesConfiguration
      */
     public function __construct(
-        FlashBagInterface $flashBag,
+        /* SessionInterface */ $flashBag,
         LoggerInterface $logger,
         SlugifierInterface $slugifier,
         RequestStack $requestStack,
         DomainConfigurationInterface $domainConfiguration,
         PagesConfiguration $pagesConfiguration
     ) {
         $this->flashBag = $flashBag;
+
+        if ($flashBag instanceof FlashBagInterface) {
+            @trigger_error('Passing the "@session.flash_bag" service as first argument is deprecated since KunstmaanNodeBundle 5.5 and will be replaced by the session in KunstmaanNodeBundle 6.0. Inject the "@session" service instead.', E_USER_DEPRECATED);
```

Comment:
```suggestion
            @trigger_error('Passing the "@session.flash_bag" service as first argument is deprecated since KunstmaanNodeBundle 5.6 and will be replaced by the session in KunstmaanNodeBundle 6.0. Inject the "@session" service instead.', E_USER_DEPRECATED);
```